src/ert/ensemble_evaluator/evaluator_connection_info.py

```diff
@@ -5,18 +5,5 @@
 class EvaluatorConnectionInfo:
     """Read only server-info"""
 
-    url: str
-    cert: str | bytes | None = None
+    router_uri: str
```

Comment:
Should we include `router` in this instance variable? This requires the reader to know the dealer/router pattern, and couples to the implementation very tightly.

Reply:
This knowledge about router-dealer needs to be propagated regardless. I'd like to keep it explicit that this is to connect to router and nothing else :)